version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.14-SNAPSHOT"
+version in ThisBuild := "0.0.17-SNAPSHOT"
```

Comment:
Is this version bump needed? version 13 is the latest version in the S3 repo?

Reply:
No, I think that was me trying to force a new local version, I've reverted it.